src/topojson/topojson.ml

```diff
@@ -495,17 +507,19 @@ module Make (J : Intf.Json) = struct
                     (decode_or_err (J.to_array (decode_or_err J.to_float)))))
               arcs
           in
+          let transform = transform_of_json json in
           let fm = foreign_members_of_json json in
-          Ok { objects = geometries; arcs; foreign_members = fm }
+          Ok { objects = geometries; arcs; transform; foreign_members = fm }
       | _, _ -> Error (`Msg "No objects and/or arcs field in Topology object!")
 
-    let to_json ?bbox { objects; arcs; foreign_members } =
+    let to_json ?bbox { objects; arcs; foreign_members; transform } =
       J.obj
         ([
            ("type", J.string "Topology");
            ( "objects",
              J.obj (List.map (fun (k, v) -> (k, Geometry.to_json v)) objects) );
            ("arcs", J.array (J.array (J.array J.float)) arcs);
+           ("transform", J.obj [ "scale", J.array J.float [| fst t.scale; snd t.scale |]] );
```

Comment:
```suggestion
           ("transform", J.obj [ "scale", J.array J.float [| fst transform.scale; snd transform.scale |]] );
```

Reply:
Note as well that the transform is optional so we need to add it to the list if it exists otherwise we don't.